frontend/Profile/NotificationCommentWrapper.svelte

```diff
@@ -1,8 +1,132 @@
 <script>
     import Comment from "../Discussion/Comment.svelte";
+    import CommentEditor from "../Discussion/CommentEditor.svelte";
+    import { sendMessage } from "../Stores/AlertStore";
     import { userList } from "../Stores/UserlistSubscriber";
     let users = $userList;
     export let supportData;
+    let replying = false;
+    let fetching = false;
+    let replyUser = "";
+
+    const newCommentTemplate = {
+        id: "",
+        message: "",
+        release: "",
+        reactions: {},
+        mentions: [],
+        user_id: "",
+        release_id: "",
+        test_run_id: supportData.test_run_id,
+        posted_at: new Date(),
+    };
+
+    let newCommentBody;
+
+    const handleReplyButton = function (e) {
+        newCommentBody = Object.assign({}, newCommentTemplate);
+        let authorId = e.detail.author;
+        let authorUsername = users?.[authorId]?.username;
+        replyUser = authorUsername;
+        let quotedMessage = e.detail.message
+            .trim()
+            .split("\n")
+            .map((val) => `> ${val}`)
+            .join("\n");
+        newCommentBody.message = `${authorUsername ? `@${authorUsername} posted:` : ""}${
+            newCommentBody.message
+        }\n${quotedMessage}\n`;
+        replying = true;
+    };
+
+    const handleCommentSubmit = async function (e) {
+        let commentBody = e.detail;
+        fetching = true;
+        try {
+            let apiResponse = await fetch(
+                `/api/v1/test/${supportData.test_id}/run/${supportData.test_run_id}/comments/submit`,
```

Comment:
why test_id is needed?

Reply:
The `test_id` is needed to resolve the plugin to which the `test_run_id` belongs to, without it we would have to query each plugin table - SCT, driver matrix, sirenada - to be able to find the run.

Reply:
why we care about plugin in context of comments?

Reply:
To build the url for the user to click inside notification in case they want to visit the run